docs/api-irma-server.md

```diff
@@ -58,29 +66,36 @@ Each session starts in the `"INITIALIZED"` [session status](#get-session-token-s
 
 Cancel the session: set the [session status](#get-session-token-status) to `"CANCELLED"`.
 
+> There is also a [variant of this endpoint](#delete-irma-session-clienttoken) for client tokens (the token in a `sessionPtr`).
```

Comment:
I'd like to remain the reference to the other endpoint, since we get a lot of comments of people not understanding the difference between both endpoints (because of the name similarity, the only difference is the extra `/irma` prefix). They get confused by the different tokens, so I think it would be wise to explicitly point out this difference.

Maybe something like:
`There is also a [variant of this endpoint](#delete-irma-session-clienttoken) for frontends (and IRMA apps) using client tokens (the token in a `sessionPtr`).`

Reply:
I already updated it to such a line as I proposed (a little bit different, I better describe what part of the sessionPtr is the client token)